assign_rights/assemble.py

```diff
@@ -6,9 +8,12 @@
 class RightsAssembler(object):
     """docstring for RightsCalculator"""
 
-    def retrieve_rights(self):
+    def retrieve_rights(self, item, shell_list):
         """docstring for retrieve_rights"""
-    pass
+        identifier = item.get("identifier")
```

Comment:
See my comment on the tests data below - `item` _may_ in this case be an identifier.